README.md

```diff
@@ -266,6 +266,15 @@ You can also define your inputs in external files. Adapt your `.kitchen.yml` to
         - test/integration/profile-attribute.yml
 ```
 
+### Use inspec plugins
+
+By default, the verified does not load Inspec plugins like additional inputs. You can activate loading the same plugins as on normal Inspec invocations:
```

Comment:
I suggest just a couple of small edits here - 

...`the verifier does not load Inspec plugins such as additional input plugins`...

Reply:
Good one, that typo is especially annoying :)